client/locations-list-map.js

```diff
@@ -1524,14 +1540,22 @@ function initMap(data, filters) {
       const currentLat = mapBounds.getCenter().lat();
       const currentLng = mapBounds.getCenter().lng();
 
-      if (currentLat !== gCurrentViewportCenter.lat || currentLng !== gCurrentViewportCenter.lng) {
-        refreshList(data, filters);
+      if (showList) {
+        // When "changing" to initial bounds, the location list is already in sync,
+        //  *unless* the URL specified coordinates.
+        if ((gCurrentViewportCenter.lat && gCurrentViewportCenter.lat !== currentLat)
+          || (gCurrentViewportCenter.lng && gCurrentViewportCenter.lng !== currentLng)
+          || searchParams.coords) {
```

Comment:
what about if the URL specified a search query `q=`?

Reply:
also, can this conditional just be:

    gCurrentViewportCenter.lat !== currentLat || gCurrentViewportCenter.lng !== currentLng || searchParams.coords

seems like if `gCurrentViewportCenter.lat` is undefined or null or otherwise falsey then the condition will be false without having the additional clause?

Reply:
No, the first time this runs, `gCurrentViewpointCenter` has its initial value `{}`, so `.lat` is undefined and is `!== currentLat`. That's what triggered the redundant call to `refreshList` that started me on this.

I did try changing `gCurrentViewpointCenter` to be `null` initially, but that caused some other problem that I've forgotten, so I went with this.